ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/provision/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-provision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-provision
```

Comment:
If this chain is only used for the next integration, why not move the steps to `cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-hypershift-provision`?

Reply:
this workflow `cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt` still uses this step

Reply:
Is `cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt` workflow will be covered in ci or some other place is using it?

Reply:
In the future, will need it, ecosystem, and cnv team

Reply:
okay.